wp-seo-main.php

```diff
@@ -315,7 +315,18 @@ function wpseo_admin_init() {
 	add_action( 'admin_init', 'yoast_wpseo_missing_filter', 1 );
 }
 
-if ( ( ! defined( 'WP_INSTALLING' ) || WP_INSTALLING === false ) && ( $spl_autoload_exists && $filter_exists ) ) {
+if ( ! function_exists( 'wp_installing' ) ) {
+	/**
+	 * We need to define wp_installing in WordPress versions older than 4.4
+	 *
+	 * @return bool
+	 */
+	function wp_installing() {
+		return defined( 'WP_INSTALLING' );
```

Comment:
Why not use the complete functionality from WordPress here?
We assume here we are the only ones who will try to use this function if it exists, does not feel safe.